code/modules/admin/verbs/selectequipment.dm

```diff
@@ -77,36 +77,30 @@
  *
  * Args:
  * * category (string) - The tab it will be under
+ *
  * * identifier (typepath or ref) - This will sent this back to ui_act to preview or spawn in an outfit.
- * *								Must be unique between all entries.
+ * * Must be unique between all entries.
+ *
  * * name (string) - Will be the text on the button
- * * priority (int)(optional) - Default SE_PRIORITY_DEFAULT, SE_PRIORITY_FAVORITE for favorites,
- * *							SE_PRIORITY_TOP for buttons that should always be on the top of the list
+ *
+ * * priority (bool)(optional) - If True, the UI will sort the entry to the top, right below favorites.
+ *
  * * custom_entry (bool)(optional) - Send the identifier with a "ref" keyword instead of "path",
- * *								 for the UI to tell apart custom outfits from normal ones.
+ * * for the UI to tell apart custom outfits from normal ones.
+ *
  * Returns (list) An outfit entry
  */
 
-#define SE_PRIORITY_DEFAULT 0
-#define SE_PRIORITY_FAVORITE 1
-#define SE_PRIORITY_TOP 2
-
-/datum/select_equipment/proc/outfit_entry(category, identifier, name, priority=SE_PRIORITY_DEFAULT, custom_entry=FALSE)
-	if(custom_entry)
-		return list("category" = category, "ref" = identifier, "name" = name, "priority" = priority)
-	return list("category" = category, "path" = identifier, "name" = name, "priority" = priority)
+/datum/select_equipment/proc/outfit_entry(category, identifier, name, priority=FALSE, custom_entry=FALSE)
+	return list("category" = category, custom_entry?"ref":"path" = identifier, "name" = name, "priority" = priority)
```

Comment:
```
list(custom_entry?"ref":"path" = identifier)
```

It work like that? Absolutely cursed.

Reply:
depressingly, yes. but this is unreadable, move it out to an if.